wordpress-mobile/WordPress-iOS
# Feature/login/wpcom password

Refs #7272

![pr-wpcom-password](https://user-images.githubusercontent.com/517257/27562841-b8b32a86-5a8b-11e7-96ed-38f8e800bde1.png)

To test:
- Login to wpcom using password: Login -> enter email -> tap Lost your password?
- Ensure login still works
- Verify design implementation

Needs review: @aerych
